src/queue.js

```diff
@@ -219,4 +219,6 @@ class Queue extends EventEmitter {
   }
 }
 
+Queue.Job = null; // Make TypeScript compiler happy. Needed because Job is an abstract class.
```

Comment:
Cool. It would be helpful if this could be pushed to npm. I can use 'yarn link' in local development but I can't test code that's running in the cluster that way.